File: Python Solution/ProjectEuler6.py
# sum1 = 0
# sum2 = 0
# for a in range(1, 101):
#     sum1 += a**2
#     sum2 += a
# print(sum2**2-sum1)
print(sum([a for a in range(1, 101)]) ** 2 -sum([b ** 2 for b in [c for c in range(1, 101)]]))

File: Java Solution/ProjectEuler6.java
public class ProjectEuler6{

    public static void main(String[] args){

        long sum1 = 0, sum2 = 0;

        for (int i = 1;i <= 100; i++){
            sum1 += i * i;
        }

        for (int i = 1;i <= 100; i++){
            sum2 += i;
        }

        sum2 = sum2 * sum2;

        System.out.println(sum2 - sum1);

    }

}